src/lib/env.js

```diff
@@ -9,13 +9,6 @@ const yargs = _yargs(hideBin(process.argv));
  * @extends Object
  * @param {number} logLevel
  * @param {boolean} logColor
- * @param {string} psqlHost
- * @param {number} psqlPort
- * @param {string} psqlUser
- * @param {string} psqlPassword
- * @param {string} psqlDatabase
- * @param {number} psqlPoolMin
- * @param {number} psqlPoolMax
  * @param {string} apiURL
```

Comment:
I dont believe fetch communicates with the API any more, this and any functionality that is using this is likely stale, re-review this usage

Reply:
API_URL is used in deploy.yml for GH secrets